swap_meet/vendor.py

```diff
@@ -0,0 +1,95 @@
+class Vendor:
+    
+    def __init__(self, inventory = None):
+        
+        #self.inventory = inventory
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+    
+    # method, adds item to self.inventory[]
+    def add(self,item):
+        if item not in self.inventory:
+            self.inventory.append(item)
+        return item 
+    
+    # returns and removes item if it is found in the inventory list
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
consider refactoring to address the edge case first
```suggestion
        if item not in self.inventory:
            return False
                
        self.inventory.remove(item)
        return item
```